misk/src/main/kotlin/misk/security/cert/X500Name.kt

```diff
@@ -12,6 +12,26 @@ class X500Name(private val components: Map<String, String>) {
 
   fun asMap() = components
 
+  override fun equals(other: Any?): Boolean {
```

Comment:
you should also be overriding hashCode

Reply:
this should probably just be a `data` class right? would get all of this for free

Reply:
will look in to making this a data class as discussed